src/notte/common/notifier/discord.py

```diff
@@ -0,0 +1,79 @@
+import discord
+from pydantic import BaseModel
+from typing_extensions import override
+
+from notte.common.agent.types import AgentResponse
+
+from .base import BaseNotifier
+
+
+class DiscordConfig(BaseModel):
+    """Configuration for Discord sending functionality."""
+
+    token: str
+    channel_id: int
+
+
+class DiscordService:
+    """Service for sending messages to Discord from Notte."""
+
+    def __init__(self, config: DiscordConfig):
+        self.config: DiscordConfig = config
+        intents = discord.Intents.default()
```

Comment:
@TonioSnowden there are still at lot of `check warning on line XX in src/notte/common/notifier/XXX` to solve in case you didn't see